PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -290,6 +289,8 @@ function Import-SecureBaseline{
                             $Value = "Error - The baseline policy text is malformed. Description should start immediately after Policy Id."
                             Write-Error "Expected description for $Id to start on line $($LineNumber+$LineAdvance)"
                         }
+                        # Description italics substitution
+                        $Value = $Value -replace '(_)(.*?)(_)', '<i>${2}</i>'
```

Comment:
This is the first of multiple description lines.  I am assuming we cannot guarentee the italics are always in the first line.  So this needs to be defered until full description is processed.  Might also consider defining a functions so we can add multiple transformations over time.  May need more than italics as we move forward.

Reply:
In my recent commit, I added the declaration of a function with this functionality.